<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>simon</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>../../../../../../../:/images/tray.png</normaloff>../../../../../../../:/images/tray.png</iconset>
  </property>
  <property name="accessibleName" >
   <string>simon</string>
  </property>
  <property name="accessibleDescription" >
   <string>simon Hauptfenster</string>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>12</y>
     <width>839</width>
     <height>672</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="lbLogo" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>100</width>
          <height>60</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap resource="others.qrc" >:/images/simon_d.png</pixmap>
        </property>
        <property name="scaledContents" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frmConnectionStatus" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="leftMargin" >
          <number>6</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>6</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbTextConnectionStatus" >
           <property name="font" >
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text" >
            <string>Nicht Verbunden</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pbConnect" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="accessibleName" >
            <string>Verbinden</string>
           </property>
           <property name="accessibleDescription" >
            <string>Verbindet zu Juliusd</string>
           </property>
           <property name="text" >
            <string>Verb&amp;inden</string>
           </property>
           <property name="icon" >
            <iconset resource="action_icons.qrc" >
             <normaloff>:/images/icons/connect-no.svg</normaloff>:/images/icons/connect-no.svg</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbAddWord" >
        <property name="accessibleName" >
         <string>Wort hinzufügen</string>
        </property>
        <property name="accessibleDescription" >
         <string>Hier kann ein neues Wort hinzugefügt werden</string>
        </property>
        <property name="text" >
         <string>Wort &amp;hinzufügen</string>
        </property>
        <property name="icon" >
         <iconset resource="action_icons.qrc" >
          <normaloff>:/images/icons/edit-add.svg</normaloff>:/images/icons/edit-add.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="shortcut" >
         <string>Ctrl+S</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbTrain" >
        <property name="accessibleName" >
         <string>Trainieren</string>
        </property>
        <property name="accessibleDescription" >
         <string>Trainieren des Sprachmodells</string>
        </property>
        <property name="text" >
         <string>&amp;Trainieren</string>
        </property>
        <property name="icon" >
         <iconset resource="other_icons.qrc" >
          <normaloff>:/images/icons/kvoctrain.svg</normaloff>:/images/icons/kvoctrain.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbWordList" >
        <property name="accessibleName" >
         <string>Wortliste</string>
        </property>
        <property name="accessibleDescription" >
         <string>Übersicht über das verwendete Vokabular</string>
        </property>
        <property name="text" >
         <string>Wort&amp;liste</string>
        </property>
        <property name="icon" >
         <iconset resource="action_icons.qrc" >
          <normaloff>:/images/icons/format-justify-fill.svg</normaloff>:/images/icons/format-justify-fill.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbRunProgram" >
        <property name="accessibleName" >
         <string>Ausführen</string>
        </property>
        <property name="accessibleDescription" >
         <string>Ausführen von definierten Aktionen</string>
        </property>
        <property name="text" >
         <string>Kommandos</string>
        </property>
        <property name="icon" >
         <iconset resource="other_icons.qrc" >
          <normaloff>:/images/icons/system-run.svg</normaloff>:/images/icons/system-run.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbSystem" >
        <property name="accessibleName" >
         <string>System</string>
        </property>
        <property name="accessibleDescription" >
         <string>Einstellungen und Konfiguration</string>
        </property>
        <property name="text" >
         <string>System</string>
        </property>
        <property name="icon" >
         <iconset resource="category_icons.qrc" >
          <normaloff>:/images/icons/preferences-system.svg</normaloff>:/images/icons/preferences-system.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QToolButton" name="pbActivision" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="accessibleName" >
         <string>Aktiviert simon</string>
        </property>
        <property name="accessibleDescription" >
         <string>Wechselt zwischen Aktiv / Inaktiv ohne die Verbindung zu trennen</string>
        </property>
        <property name="text" >
         <string>&amp;Aktivieren</string>
        </property>
        <property name="icon" >
         <iconset resource="action_icons.qrc" >
          <normaloff>:/images/icons/media-playback-start.svg</normaloff>:/images/icons/media-playback-start.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbHide" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="accessibleName" >
         <string>Verstecken</string>
        </property>
        <property name="accessibleDescription" >
         <string>Versteckt das simon-Hauptfenster</string>
        </property>
        <property name="text" >
         <string>&amp;Verstecken</string>
        </property>
        <property name="icon" >
         <iconset resource="action_icons.qrc" >
          <normaloff>:/images/icons/go-bottom.svg</normaloff>:/images/icons/go-bottom.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbCompileModel" >
        <property name="text" >
         <string>Sprachmodell generieren</string>
        </property>
        <property name="icon" >
         <iconset resource="action_icons.qrc" >
          <normaloff>:/images/icons/go-next.svg</normaloff>:/images/icons/go-next.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbKeyed" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="accessibleName" >
         <string>Systemverwaltungsmodus</string>
        </property>
        <property name="accessibleDescription" >
         <string>Wechselt zwischen den Administrations / Benutzer-Modu</string>
        </property>
        <property name="text" >
         <string>Systemverwaltungsmodus</string>
        </property>
        <property name="icon" >
         <iconset resource="action_icons.qrc" >
          <normaloff>:/images/icons/lock.svg</normaloff>:/images/icons/lock.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pbClose" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="accessibleName" >
         <string>Beenden</string>
        </property>
        <property name="accessibleDescription" >
         <string>Beendet simon</string>
        </property>
        <property name="text" >
         <string>&amp;Beenden</string>
        </property>
        <property name="icon" >
         <iconset resource="action_icons.qrc" >
          <normaloff>:/images/icons/system-log-out.svg</normaloff>:/images/icons/system-log-out.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="InlineWidgetView" name="inlineView" >
      <property name="tabPosition" >
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape" >
       <enum>QTabWidget::Rounded</enum>
      </property>
      <widget class="QWidget" name="tbWelcome" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>819</width>
         <height>516</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Start</string>
       </attribute>
       <attribute name="icon" >
        <iconset resource="others.qrc" >
         <normaloff>:/images/tray.png</normaloff>:/images/tray.png</iconset>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QLabel" name="lbWelcomeDesc" >
          <property name="text" >
           <string/>
          </property>
          <property name="pixmap" >
           <pixmap resource="banners.qrc" >:/images/home.png</pixmap>
          </property>
          <property name="alignment" >
           <set>Qt::AlignJustify|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="tbStatus" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>684</y>
     <width>839</width>
     <height>12</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Globale Aktionen</string>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="tbLogo" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>10</width>
     <height>12</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Logo</string>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="tbModules" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>829</width>
     <height>12</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Module</string>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InlineWidgetView</class>
   <extends>QTabWidget</extends>
   <header>inlinewidgetview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="action_icons.qrc" />
  <include location="application_icons.qrc" />
  <include location="banners.qrc" />
  <include location="category_icons.qrc" />
  <include location="other_icons.qrc" />
  <include location="others.qrc" />
 </resources>
 <connections/>
</ui>
